### PR TITLE
Improve documentation language

### PR DESCRIPTION
**Pull request by strugee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,6 @@ To edit source
 
 ## Using
 
-Protip: Don't
+ProTip™: Don't
```
Comment (shelvacu) [inline README.md:17]: Only registered trademarks are accepted.

**Review by shelvacu — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (strugee) [reply README.md:17]: @shelvacu this project is a disgrace, how dare you not merge my feature!!!!!!!!!1!!!1!!!!! I will be forking ASAP

You suck

Comment (RobertCochran) [reply README.md:17]: :'(

Comment (shelvacu) [reply README.md:17]: fork you too @strugee

**Review by Riven-Spell — APPROVED** [review]

**Comment by shelvacu** [comment]
Whoops, my finger slipped. Twice.

**Comment by strugee** [comment]
Victory™®©

**Pull request merged**